app/views/metadata_presenter/component/_email.html.erb

```diff
@@ -9,3 +9,12 @@
     spellcheck: "false",
     autocomplete: "email"
 %>
+
+<% if is_confirmation_email_question? %>
+  <div class ="govuk-warning-text">
+    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+    <strong class="govuk-warning-text__text">
```

Comment:
I think a space is needed here between the span and the text? I think we use the govuk margin class elsewhere,  add `govuk-!-margin-right-2` to the span class? It's not present in the example but in the screenshot the icon looks too close to the text :D

Reply:
Sorry this was solved at some point. This is how it looks like now: ![image](https://github.com/ministryofjustice/fb-metadata-presenter/assets/26165112/4d9981fb-5988-48d7-a62d-b7b13daad800)